src/sentry/migrations/0284_metrics_indexer_alter_seq.py

```diff
@@ -27,16 +27,4 @@ class Migration(CheckedMigration):
         ("sentry", "0283_extend_externalissue_key"),
     ]
 
-    operations = [
```

Comment:
Why this ?

Reply:
raw sql statements mentioning sequence don't work. We cannot run this migration as-is, so I think we would need to edit it to skip doing that for spanner, or create a new table name etc for spanner where this migration doesn't run.